.lintstagedrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  "*.*": "prettier '**/*' --write --ignore-unknown",
```

Comment:
What extensions do we expect here?

Reply:
All extensions prettier can support
I don't understand the drawbacks of this

Reply:
OK, we can exclude js/ts files cuz they have their own scrip.

Reply:
whats about .json files or .md or ...
It is true that there may not be such files in our project at all, but I think handling them is not bad

Reply:
whats about .json files or .md or ...
It is true that there may not be such files in our project at all, but I think handling them is not bad